data/dataexchange/src/main/java/tech/pegasys/teku/data/SlashingProtectionImporter.java

```diff
@@ -81,6 +81,9 @@ public void initialise(final File inputFile) throws IOException {
 
     } catch (JsonMappingException e) {
       String cause = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
+      if (cause.contains("\"interchange_format\"")) {
+        cause = "Required version is " + INTERCHANGE_VERSION;
+      }
```

Comment:
This doesn't seem ideal. If the only change is removing this field, can we support both version 4 and 5 and just mark the field optional?